Model/Source/DefaultOptions.php

```diff
@@ -314,7 +309,7 @@ public function getPackageTypeName(): string
      * TODO: In the future, when multiple carriers will be available for Rest of World shipments, replace PostNL with a setting for default carrier
      *
      * @return \MyParcelNL\Sdk\src\Model\Carrier\AbstractCarrier
-     * @throws \Exception
```

Comment:
dit moet niet, in docs gebruiken we FQCNs

Reply:
Ik heb het aangepast